Submitted By: Joe Ciccone <jciccone@gmail.com>
Date: 2008-06-28
Initial Package Version: 5.8.8
Upstream Status: Unknown
Origin: http://trac.cross-lfs.org/ticket/160
Description: Removes the only asm/page.h inclusion from perl.

diff -Naur perl-5.8.8.orig/ext/IPC/SysV/SysV.xs perl-5.8.8/ext/IPC/SysV/SysV.xs
--- perl-5.8.8.orig/ext/IPC/SysV/SysV.xs	2001-06-30 14:46:07.000000000 -0400
+++ perl-5.8.8/ext/IPC/SysV/SysV.xs	2008-06-28 14:03:42.804556568 -0400
@@ -3,9 +3,6 @@
 #include "XSUB.h"
 
 #include <sys/types.h>
-#ifdef __linux__
-#   include <asm/page.h>
-#endif
 #if defined(HAS_MSG) || defined(HAS_SEM) || defined(HAS_SHM)
 #ifndef HAS_SEM
 #   include <sys/ipc.h>
